README.md

```diff
@@ -15,15 +21,64 @@ $ pip install .
 ```
 
 3) [Optional] Enable bash completion
+for green-cli:
 ```
 $ eval "$(_GREEN_CLI_COMPLETE=source green-cli)"
+```
+
+for green-liquid-cli:
+```
 $ eval "$(_GREEN_LIQUID_CLI_COMPLETE=source green-liquid-cli)"
 ```
 
-Example usage (testnet):
+# Hardware wallet support (via hwi) [BETA]
+
+To enable hardware wallet support (via the `--auth hardware` option)
+additional dependencies must be installed from requirements-hwi.txt.
+
+1) Install libudev and libusb. This is platform specific but for debian:
```

Comment:
not: debian-based systems (this works for ubuntu as well for example)

Reply:
done, thanks